examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -3,44 +3,45 @@
 Adjustment for chance in clustering performance evaluation
 ==========================================================
 
-The following plots demonstrate the impact of the number of clusters and
-number of samples on various clustering performance evaluation metrics.
+The following plots demonstrate the impact of the number of clusters and number
+of samples on various clustering performance evaluation metrics.
 
-Non-adjusted measures such as the V-Measure show a dependency between
-the number of clusters and the number of samples: the mean V-Measure
-of random labeling increases significantly as the number of clusters is
-closer to the total number of samples used to compute the measure.
+Non-adjusted measures such as the V-Measure show a dependency between the number
+of clusters and the number of samples: the mean V-Measure of random labeling
+increases significantly as the number of clusters is closer to the total number
+of samples used to compute the measure.
 
-Adjusted for chance measure such as ARI display some random variations
-centered around a mean score of 0.0 for any number of samples and
-clusters.
+Adjusted for chance measure such as ARI display some random variations centered
+around a mean score of 0.0 for any number of samples and clusters.
```

Comment:
This two paragraphs should be moved after the first plot to explain what is interesting about it.

Reply:
In particular, each plot should have a paragraph after it to describe what can be seen in it.

Here the conclusions are mostly the same for both experiments, so for the second plot we can just add a sentence that states that we observe similar results as for the first experiment: adjusted for chance metrics stay constantly near zero while other metrics tend to get larger with finer-grained labelings.